<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>540</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>128</red>
        <green>128</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>57</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>128</red>
        <green>128</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>57</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>128</red>
        <green>128</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>148</green>
        <blue>148</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>57</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>57</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>203</width>
      <height>641</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton { 
	image: url(:/res/Type 3/Mission_Planning_Button_Type_3.png);
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
QPushButton:hover{
 	background-color: #505050
}
 QPushButton:pressed {
     background-color: #000000
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton { 
	image: url(:/res/Type 3/Mission_Execution_Button_Type_3.png);
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
QPushButton:hover{
 	background-color: #505050
}
 QPushButton:pressed {
     background-color: #000000
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>102</red>
             <green>102</green>
             <blue>102</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>102</red>
             <green>102</green>
             <blue>102</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>102</red>
             <green>102</green>
             <blue>102</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>67</red>
             <green>74</green>
             <blue>84</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton { 
	image: url(:/res/Type 3/Mission_Recap_Button_Type_3.png);
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
QPushButton:hover{
 	background-color: #505050
}
 QPushButton:pressed {
     background-color: #000000
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton { 
	image: url(:/res/Type 3/Tutorial_Button_Type_3.png);
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
QPushButton:hover{
 	background-color: #505050
}
 QPushButton:pressed {
     background-color: #000000
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton { 
	image: url(:/res/images/UAVMissionPlanningPreview.jpg);
background-color: &quot;#FFFFFF&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton { 
	image: url(:/res/images/UAVMissionRecapTest.png);
background-color: &quot;#FFFFFF&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton { 
	image: url(:/res/images/UAVMissionExecutionTest.jpg);
background-color: &quot;#FFFFFF&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>360</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton { 
	image: url(:/res/images/UAVLogo.png);
background-color: &quot;#FFFFFF&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Missions">
     <property name="title">
      <string>Recent Missions</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="menuRecent_Missions"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionTest"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMap_Planning"/>
    <addaction name="actionMission_Execution"/>
    <addaction name="actionMission_Recap"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTools"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHELP_ME"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuTest"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New Mission</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Mission</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Mission</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import Mission</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export Mission</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionMap_Planning">
   <property name="text">
    <string>Map Planning</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionMission_Execution">
   <property name="text">
    <string>Mission Execution</string>
   </property>
  </action>
  <action name="actionMission_Recap">
   <property name="text">
    <string>Mission Recap</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionHELP_ME">
   <property name="text">
    <string>HELP ME</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openTutorial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>missionPlanningClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>missionExecutionClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>948</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>missionRecapClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>946</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>missionPlanningClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>missionExecutionClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1110</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>missionRecapClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openOptions()</slot>
  <slot>openTutorial()</slot>
  <slot>missionPlanningClicked()</slot>
  <slot>missionExecutionClicked()</slot>
  <slot>missionRecapClicked()</slot>
 </slots>
</ui>
